test/arbitrary.test.ts

```diff
@@ -291,7 +291,7 @@ describe('Arbitrary tests', () => {
   describe('Chained Arbitraries', () => {
     it('should allow the creation of array with size based on an integer arbitrary', () => {
       expect(
-        fc.integer(2, 2).chain(i => fc.array(fc.constant(i), i, i)).pick()!.value
+        fc.integer(2, 2).chain(i => fc.array(fc.constant(i), i, i)).pick(Math.random)!.value
```

Comment:
Why are we changing the RNG in this particular test?

Reply:
Because fc.array returns an object of type Arbitrary and since the pick method in Arbitrary is marked as abstract it cannot have a default value for its parameters. A solution is change the code for the pick method in the Arbitrary class to:

pick(generator: () => number = Math.random): FluentPick<A> | undefined { return }

Which version do you prefer?